deps/libgit2.mk

```diff
@@ -40,6 +40,10 @@ ifeq ($(OS),Linux)
 LIBGIT2_OPTS += -DUSE_OPENSSL=OFF -DUSE_MBEDTLS=ON -DCMAKE_INSTALL_RPATH="\$$ORIGIN"
 endif
 
+ifeq ($(OS),FreeBSD)
+LIBGIT2_OPTS += -DCMAKE_INSTALL_RPATH="\$$ORIGIN"
```

Comment:
Yes, libgit2 will link in the system's libssl by default on FreeBSD. Setting the same options as Linux here (`-DUSE_OPENSSL=OFF -DUSE_MBEDTLS=ON`) causes a fatal error on Julia startup.